config/default.yml

```diff
@@ -47,6 +47,11 @@ linters:
   ImplicitDiv:
     enabled: true
 
+  Indentation:
+    enabled: true
+    character: space # or tab
```

Comment:
Also added a comment here and in the README explaining that width is ignored in `tab`-mode.